index.html

```diff
@@ -122,6 +159,161 @@ <h2>Methods</h2>
       </dl>
     </section>
   </section>
+  <section id="rtcrtptransceiver-interface">
+    <h3>
+      <dfn>RTCRtpTransceiver</dfn> extensions
+    </h3>
+    <p>
+      The <code><a href="https://w3c.github.io/webrtc-pc/#dom-rtcrtptransceiver">RTCRtpTransceiver</a></code>
+      interface is defined in [[WEBRTC]]. This document extends that interface
+      by adding an additional method and attribute in order to control negotiation of
+      RTP header extensions.
+    </p>
+    <pre class="idl">
+partial dictionary RTCRtpHeaderExtensionCapability {
+  RTCRtpTransceiverDirection direction = "sendrecv";
+};
+      
+partial interface RTCRtpTransceiver {
+  void setOfferedRtpHeaderExtensions(
+    sequence&lt;RTCRtpHeaderExtensionCapability&gt; headerExtensionsToOffer);
+  readonly attribute FrozenArray&lt;RTCRtpHeaderExtensionCapability&gt; headerExtensionsNegotiated;
+};
+    </pre>
+    <p>
+      Let [[\HeaderExtensionsToOffer]] be an internal slot of the RTCRtpTransceiver,
+      initialized to the platform-specific initial set of RTP header extensions that
+      is offered on an initial CreateOffer. This list MUST include all header extensions
+      that are marked Mandatory to Implement in the underlying protocol. The direction
+      attribute for all extensions that are mandatory to use MUST be initialized to
+      "sendrecv".
+    </p>
+    <p class="note">The list of header extensions that MUST/SHOULD be supported is listed in
+      draft-ietf-rtcweb-rtp-usage, section 5.2. The "mid" extension is mandatory to use when
+      BUNDLE is in use.
+    </p>
+    <p>
+      Let [[\HeaderExtensionsNegotiated]] be an internal slot of the RTCRtpTransceiver,
+      initialized to an empty list.
+    </p>
+    <p>
+      In the algorithms for generating initial offers in JSEP section 5.1.1,
+      read "for each supported RTP header extension" as "For each RTP header extension
+      listed in [[\HeaderExtensionsToOffer]] where direction is not "stopped"".
+    </p>
+    <p class="note">
+      Since JSEP does not know about WebRTC internal slots, merging this change requires
+      more work on a JSEP revision.
+    </p>
+    <p>
+      In the algorithm to <a>set an SDP description</a>, add a bullet in the sections
+      for processing answer and pranswer (bullet 3.2.8.1.7 and 3.2.8.2.6):
+      <ul>
+        <li>Set the transceiver's [[\HeaderExtensionsNegotiated]] slot to the list of
+          RTP header extensins from the answer.
```

Comment:
There's already a comment in the webrtc spec about which directions apply to sender vs receiver.
We can say the same thing here - the details of when to send various extensions are either in some IETF spec or left to the implementation, so what we can say here is that we enable sending (enabled=true on the sender).
The receiver has no choice about whether to receive or not, but it can choose to act only on those that have been negotiated (discard any others).